Sources/SQLite/Typed/CustomFunctions.swift

```diff
@@ -39,76 +39,76 @@ public extension Connection {
     ///     The assigned types must be explicit.
     ///
     /// - Returns: A closure returning an SQL expression to call the function.
-    public func createFunction<Z : Value>(_ function: String, deterministic: Bool = false, _ block: @escaping () -> Z) throws -> (() -> Expression<Z>) {
+    func createFunction<Z : Value>(_ function: String, deterministic: Bool = false, _ block: @escaping () -> Z) throws -> (() -> Expression<Z>) {
```

Comment:
I've been away from the repo for awhile, but why were these made `internal`?

Reply:
Because the extension is explicitly marked `public`, the methods receive the same access level unless otherwise specified. Xcode 10.2 provides a build warning for redundant access levels.

> Alternatively, you can mark an extension with an explicit access-level modifier (for example, private extension) to set a new default access level for all members defined within the extension. This new default can still be overridden within the extension for individual type [members.](https://docs.swift.org/swift-book/LanguageGuide/AccessControl.html)

![Screen Shot 2019-04-19 at 10 47 22 AM](https://user-images.githubusercontent.com/49003548/56434155-a71cb580-6290-11e9-97e7-868dc1f104fd.png)

Reply:
Long story short, those methods should still be public.

Reply:
Ah I missed the `public extension`. Makes sense!